04-reporting_bugs.Rmd

```diff
@@ -74,6 +74,17 @@ Once you confirm a bug exists, you need to submit a bug report so that it gets f
 
 For packages that are not maintained by the R-Core, the bug reports can be submitted at, perhaps, an issues tracker url on GitHub/GitLab/R-Forge. To find if such an issues tracker is available, you can look at the package `DESCRIPTION` file first (e.g. using `packageDescription("package_name")`) to check if a url is provided in the `BugReports` field. If that is not available, then the package maintainer can be contacted (using `maintainer("package_name")`). In R running the function `bug.report(package = "package_name")` shall direct you to either the GitHub issue tracker of the package, or to the bug tracking web page, or towards composing an e-mail to the package maintainer. This function `bug.report` is disabled in RStudio, by default. However, if you use `utils::bug.report(package = "package_name")` then it works on RStudio as well. Please ensure that the bug is not already reported or fixed before reporting it in any of the ways suggested above.
 
+### Bug in a documentation website managed by the R-Core team {#docWebsRCoreBug}
+
+The R-Core team manages a list of website connected to the R project. If you noticed a bug or an issue which you would like to report, please [see posting guide](https://www.r-project.org/posting-guide.html#which_list) and then submit the issue report by sending an email to R-devel@R-project.org
+
+The list of websites managed by the R-Core team: 
+- [R FAQ](https://cran.r-project.org/doc/FAQ/R-FAQ.html)
+- [R for Windows FAQ](https://cran.r-project.org/bin/windows/base/rw-FAQ.html)
+- [R for macOS FAQ](https://cran.r-project.org/bin/macosx/RMacOSX-FAQ.html)
+- [The R Project website](https://www.r-project.org/)
+- [The Comprehensive R Archive Network](https://cran.r-project.org/)
```

Comment:
I found a few different emails on the sites, think an updated version of this with those included could look like:

```
The R-Core team also maintain a selection of websites that each have their own ways for reporting issues outside of Bugzilla. Have a read through their [posting guide](https://www.r-project.org/posting-guide.html#which_list) and then follow the instructions below to give feedback to the maintainers.

1. [R FAQ](https://CRAN.R-project.org/doc/manuals/R-FAQ.html) contains answers to some of the most frequently asked questions about R. Feedback on this site should be provided to [R-devel@R-project.org](mailto:R-devel@R-project.org).

2. [R for Windows FAQ](https://cran.r-project.org/bin/windows/base/rw-FAQ.html) contains answers specific to the Windows port of R. Comments on the FAQ site can be sent to [R-windows@R-project.org](mailto:R-windows@R-project.org).

3. [R for macOS FAQ](https://cran.r-project.org/bin/macosx/RMacOSX-FAQ.html) contains a user guide to the R.APP version of R, and information on using R on macOS, supplementing the main R manuals. To submit requests, ask questions or give feedback, subscribe to the [R-SIG-Mac mailing list](https://stat.ethz.ch/mailman/listinfo/r-sig-mac), the Special Interest Group on Mac ports of R, and then post your message to the mailing list by emailing [r-sig-mac@r-project.org](mailto:r-sig-mac@r-project.org).

4. Queries about [The Comprehensive R Archive Network website](https://www.r-project.org/) should be sent to [webmaster@r-project.org](mailto:webmaster@r-project.org).

5. Queries about the [CRAN website](https://cran.r-project.org/) should be sent to [cran-sysadmin@r-project.org](mailto:cran-sysadmin@r-project.org).
```